### PR TITLE
Replace footer with drawer menu

### LINKED ISSUE #363: Replace footer with Navigation drawer
- remove the footer component
- use Navigation drawer as already in mobile app to display footer elements
- display it on the right part of the page
- remove language picker (since now displayed within the menu)

### PR DESCRIPTION
**Pull request by MattiaPrimavera**
- remove footer from all pages
- move lang picker into the `NavigationDrawer` only
- replace `react-bootstrap` component `Navbar` with `AppBar` from `material-ui`
- add missing footer entries into `SwipeableTemporaryDrawer`

Fix #363

### To do in separate issues
- [ ] NewsletterModal opening needs being handled a different way
- [ ] Replace "default" menu entries icons
- [ ] Style links menu elements
- [ ] Fix menu logo (large screen, otherwise it seems fine)
- [ ] Place logo with longline into `AppBar`

### CODE REVIEW

File: src/components/App/AppBar/AppBar.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                <MaterialAppBar color="primary" position="static">
+                    <Toolbar>
+                        <Typography
+                            color="primary"
+                            variant="h6"
+                            className={classes.title}>
+                            Instant Visio
+                        </Typography>
+                        <Button color="primary">Join Visio</Button>
+                        <Button color="primary">Login</Button>
```
Comment (ntchou) [inline src/components/App/AppBar/AppBar.tsx:46]: Missing translation for join and login button

**Comment by HugoGresse** [comment]
are we sure to remove footer everywhere? I agree this is useless on mobile apps but not on web where user are used to have it to find credit/about/legal mentions/contact etc

**Comment by MattiaPrimavera** [comment]
Yes, design specs confirmed

**Comment by HugoGresse** [comment]
discussion there : https://instantvisio.slack.com/archives/C01EW0X2FJQ/p1608120406006000

**Review by HugoGresse — APPROVED** [review]

**Pull request merged**